src/mpid/ch4/subconfigure.m4

```diff
@@ -348,19 +348,15 @@ if test "$ch4_nets_array_sz" = "1" && test "$enable_ch4_netmod_inline" = "yes" &
    PAC_APPEND_FLAG([-DNETMOD_INLINE=__netmod_inline_${ch4_netmods}__], [CPPFLAGS])
 fi
 
-
-AC_ARG_ENABLE(ch4-shm,
-    [--enable-ch4-shm=level:module
-       Control whether CH4 shared memory is built and/or used. Default
-       shm level depends on selected netmod(s). (OFI=exclusive, UCX=no).
+AC_ARG_ENABLE(ch4-direct,
+    [--enable-ch4-direct
+       Defines the direct communication routine used in CH4 device
        level:
-         no        - Do not build or use CH4 shared memory.
-         yes       - Build CH4 shared memory, but do not use it by default (Your chosen netmod must provide it).
-         exclusive - Build and exclusively use CH4 shared memory.
-       module-list(optional).  comma separated list of shared memory modules:
-         posix     - POSIX shared memory implementation
-    ],,enable_ch4_shm=default)
-
+         netmod     - Directly transfer data through the chosen netmode (default)
+         auto       - The CH4 device controls whether transfer data through netmod 
```

Comment:
The whitespace checker script doesn't check non-C files, so it missed this (and a few other lines in this file) that have trailing whitespace.

Reply:
Thanks for catching this issue. I just pushed the cleanup.